IFC4x3/Templates/Product Shape/Product Geometric Representation/Body Geometry/Body AdvancedSweptSolid Geometry/Body SectionedSolidHorizontal/DocTemplateDefinition.xml

```diff
@@ -16,18 +16,7 @@
 										</DocModelRuleAttribute>
 										<DocModelRuleAttribute CardinalityMin="1" CardinalityMax="1" Name="RepresentationIdentifier" Identification="Identifier">
 											<Rules>
-												<DocModelRuleEntity id="IfcLabel_2nTsBx4yb0wfI4IH_ymbNg" Name="IfcLabel" UniqueId="b17762fb-13c9-40ea-9484-491fbcc255ea">
-													<Rules>
-														<DocModelRuleConstraint>
-															<Expression xsi:type="DocOpStatement" Operation="compareequal">
-																<Reference xsi:type="DocOpReference">
-																	<EntityRule xsi:type="DocModelRuleEntity" xsi:nil="true" href="IfcLabel_2nTsBx4yb0wfI4IH_ymbNg" />
-																</Reference>
-																<Value xsi:type="DocOpLiteral" Operation="loadstring" Literal="Body" />
-															</Expression>
-														</DocModelRuleConstraint>
-													</Rules>
-												</DocModelRuleEntity>
+												<DocModelRuleEntity Name="IfcLabel" UniqueId="b17762fb-13c9-40ea-9484-491fbcc255ea" />
```

Comment:
```suggestion
												<DocModelRuleEntity Name="IfcLabel" href="IfcLabel_2fgI0VXve2tV6eMzYUyfa9"/>
```
this will reference and load the rule correctly in IfcDoc